Automattic/wp-calypso
# Add MultipleChoiceQuestion Component

#### Changes proposed in this Pull Request

<img width="592" alt="Screen Shot 2019-03-11 at 10 34 47 AM" src="https://user-images.githubusercontent.com/2810519/54144617-512f3500-43e9-11e9-9ea6-c8821da4f527.png">

* New MultipleChoiceQuestion Component that renders multi choice questions. It should:
   - take a list of questions
   - scramble the order of the indicated ones,( example: scramble 1-3, 4 is always last )
   - Have options that open a text field for more input
   - use a callback to pass the information on changes

#### Testing instructions

1. Navigate to `/devdocs/design/multiple-choice-question` on your live or local branch
1. Verify you can only select one question at a time
1. Verify that the "Selected Answer is:" shows the correct question ( it will be a _kebab case_ id, not the full question )
1. Verify that the `Ravenous Rhinos` and `Something Else` options present a text input when selected
1. Verify that the "Answer Text is:" shows the text in the input
1. Verify that the text input only shows if one question is selected
1. Verify that each text input saves its text across being selected/unselected
1. Refresh the page and verify that the order of the first three questions changes
1. Run `npm run test-client client/components/multiple-choice-question/` and verify that the tests pass
